apps/main/src/llamalend/features/market-list/UserPositionTabs.tsx

```diff
@@ -1,27 +1,71 @@
-import { useState } from 'react'
-import type { LlamaMarketsResult } from '@/llamalend/entities/llama-markets'
+import { useMemo, useState } from 'react'
 import Stack from '@mui/material/Stack'
 import { t } from '@ui-kit/lib/i18n'
 import { TabsSwitcher, type TabOption } from '@ui-kit/shared/ui/TabsSwitcher'
 import { MarketRateType } from '@ui-kit/types/market'
+import { LlamaMonitorBotButton } from './LlamaMonitorBotButton'
 import { UserPositionsTable, type UserPositionsTableProps } from './UserPositionsTable'
 
-const tabs: TabOption<MarketRateType>[] = [
-  { value: MarketRateType.Borrow, label: t`Borrow` },
-  { value: MarketRateType.Supply, label: t`Supply` },
-]
+export const UserPositionsTabs = (props: Omit<UserPositionsTableProps, 'tab' | 'openPositionsByMarketType'>) => {
+  // Calculate total positions across all markets (independent of filters)
+  const openPositionsCount = useMemo((): Record<MarketRateType, number | undefined> => {
+    const markets = props.result?.markets
+    return {
+      [MarketRateType.Borrow]: markets?.filter((market) => market.userHasPositions?.[MarketRateType.Borrow]).length,
+      [MarketRateType.Supply]: markets?.filter((market) => market.userHasPositions?.[MarketRateType.Supply]).length,
+    }
+  }, [props.result?.markets])
 
-/** Show the first tab that has user positions by default, or the first tab if none are found. */
-const getDefault = (userHasPositions: LlamaMarketsResult['userHasPositions'] | undefined) =>
-  tabs.find(({ value }) => userHasPositions?.Lend[value] || userHasPositions?.Mint[value]) ?? tabs[0]
+  // Define tabs with position counts
+  const tabs: TabOption<MarketRateType>[] = useMemo(
+    () => [
+      {
+        value: MarketRateType.Borrow,
+        label: t`Borrowing`,
+        endAdornment: openPositionsCount[MarketRateType.Borrow]
+          ? String(openPositionsCount[MarketRateType.Borrow])
```

Comment:
this cast looks counter-intuitive and potentially buggy if we include JSX in `openPositionsCount`
ideally, the object should already have the type we expect i.e. `string | undefined`